src/jasmine.js

```diff
@@ -0,0 +1,54 @@
+/* eslint-env jasmine */
+import { assertions } from 'redux-actions-assertions-js';
+
+const toDispatchActions = () => {
+  return {
+    compare(action, expectedActions, done) {
+      assertions.toDispatchActions(action, expectedActions, done, done.fail);
+      return { pass: true };
+    }
+  };
+};
+
+const toNotDispatchActions = () => {
```

Comment:
Would be also nice to add a negative comparison support. See https://jasmine.github.io/2.0/custom_matcher.html#section-Custom_negative_comparators